modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,459 +1,231 @@
-* SUSE Linux Enterprise Server 11
-* Windows Server 2012
-
-Starting with this release, the following browsers are no longer supported:
-
-* Microsoft Internet Explorer
+* Ubuntu 16.04 is deprecated.
+* Debian 8 is no longer supported.
+* Java Runtime Environment(JRE) Version 8 is no longer supported.  The Analytics Service requires JRE Version 11 or later to be installed.
 
-==== Deprecated Features
+==== Deprecated and Removed Features
 
-The following functionality is deprecated or is unsupported.
+* The cbbackup, cbrestore, and cbbackupwrapper utilities are deprecated in this release.
 
-* The following values for the format parameter of the Analytics Service API are being deprecated in this release and will be removed in a future release: `application/x-adm`, `application/json`, `application/json;lossless`, `text/csv`.
-
-* Starting this release, the Analytics Service has deprecated the use of OpenJDK 8. 
-
-[#known-issues-650]
+[#known-issues-660]
 === Known Issues
 
 This section highlights some of the known issues in this release. 
 
 ==== Analytics Service
 
-[#table_knownissues_v650-analytics,cols="25,66"]
+[#table_knownissues_v660-analytics,cols="25,66"]
 |===
 | Issue | Description
 
-| https://issues.couchbase.com/browse/MB-36461[MB-36461^]
-| *Summary:* In cases where the input to IN subclause with EVERY quantifier is MISSING or NULL, Analytics and Query engines differ in behavior. The Analytics service treats MISSING or NULL input values(in this case) as equivalent to an empty array, which results in the whole `EVERY … IN …` expression returning TRUE, while the Query service returns MISSING if the input is MISSING (or NULL if the input is NULL).
-
-*Workaround:* Use the IS KNOWN predicate to test whether the IN value is not NULL/MISSING.
-
-`WHERE (x IS KNOWN) AND (EVERY y IN x SATISFIES ... END)`
+| https://issues.couchbase.com/browse/MB-40400[MB-40400^]
+| *Summary:* When using alternate addresses for remote links, at least one node in the remote cluster must have the management[SSL] port exposed, and ALL data(KV) nodes have the kv[SSL] port exposed. Failure to do so will result in a 400 (Bad Request) when creating or altering a link. 
 |===
 
-==== Cluster Manager
+==== Search Service
 
-[#table_knownissues_v650-cluster-manager,cols="25,66"]
+[#table_knownissues_v660-search,cols="25,66"]
 |===
 | Issue | Description
 
-| https://issues.couchbase.com/browse/MB-37420[MB-37420^]
-| *Summary*: Node-to-node encryption with wildcard certs in SAN entries are not supported in this release.
-|===
-
-==== Data Service
+| https://issues.couchbase.com/browse/MB-39887[MB-39887^]
+| *Summary*: Using negate(NEG) match and match_phrase queries WITHOUT the “analyzer” setting can lead to no results being returned. This is because, in a non-covering query, the context of what index to use is missing in the verification phase and the default "standard" analyzer is used instead of the "keyword" analyzer which was used in the index.  
```

Comment:
One comment on this, as it sounds a bit ambiguous ..

This can happen for non-covered queries only when either of the following are NOT specified ..

- Index name in the options
- Analyzer to use for the match query

Reply:
Thanks. Please take a look at the updated description.